underdocs-renderer/src/main/kotlin/underdocs/renderer/output/html/render/page/PageRenderer.kt

```diff
@@ -90,7 +111,21 @@ class PageRenderer(private val linker: Linker,
                                                 });
                                                })
                                         })();
-                                    """.trimIndent()))
+                                    """.trimIndent())),
+                            script()
+                                    .with(UnescapedText("""
+                                          (function () {
+                                            let checkbox = document.querySelector("#switch");
```

Comment:
Since this variable will not be reassigned, it might be a good idea to use `const` instead of `let` (see [Why most of the time should I use const instead of let in JavaScript?](https://stackoverflow.com/questions/41086633/why-most-of-the-time-should-i-use-const-instead-of-let-in-javascript) for a detailed discussion on this matter).

Another thing is, that the name of this variable is not that descriptive. There might be multiple checkboxes on the same page, hence it might not be clear which one this variable refers to. Can you please come up with a more descriptive name, please? You can find a longer winded article on this topic here: [Clean code 101 — Meaningful names and functions](https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c).

Reply:
Variable renamed and is now a const.